syntax = "proto3";

message ELG_Text {
    string PlainText = 1;
    string StructuredText = 2;
}

message ELG_Parameter {
    int32 ServiceID = 1;
}   

service RunElgService {
    rpc InitParameter(ELG_Parameter) returns(ELG_Text);
    rpc RunElgService(ELG_Text) returns(ELG_Text);
}
